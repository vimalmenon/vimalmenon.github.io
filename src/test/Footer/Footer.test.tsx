import { render } from '@testing-library/react';
import { Footer } from '@common';

describe('Footer component', () => {
  it('Snapshot Testing', () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });
});
