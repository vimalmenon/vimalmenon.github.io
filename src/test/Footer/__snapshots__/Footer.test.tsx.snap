// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component Snapshot Testing 1`] = `
<div>
  <footer
    class="css-ycjr2s"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthXl css-5a2251-MuiContainer-root"
    >
      <div
        class="MuiBox-root css-16p1w4b"
      >
        <div
          class="MuiBox-root css-1rfi0cx"
        >
          <span
            style="font-size: 14px;"
          >
            <a
              class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-bfpekr-MuiTypography-root-MuiLink-root"
              href="/about"
            >
              About
            </a>
          </span>
          <span
            style="font-size: 14px;"
          >
            <a
              class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-bfpekr-MuiTypography-root-MuiLink-root"
              href="/release-notes"
            >
              Release Notes
            </a>
          </span>
        </div>
        <div
          class="MuiBox-root css-1rfi0cx"
        >
          <span
            style="font-size: 0.8rem;"
          >
            Copyright Â© 2025
             
            <a
              class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone css-12kdy1-MuiTypography-root-MuiLink-root"
              href="https://vimalmenon.com/"
            >
              Vimal Menon V
              0.0.26
            </a>
          </span>
        </div>
      </div>
      <div
        class="MuiBox-root css-cxzwvu"
      >
        <a
          aria-label="YouTube"
          class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone css-18rqtra-MuiTypography-root-MuiLink-root"
          data-mui-internal-clone-element="true"
          href="https://www.youtube.com/@real_vimal_menon"
          target="_blank"
        >
          <button
            class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-53g0n7-MuiButtonBase-root-MuiIconButton-root"
            tabindex="0"
            type="button"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1umw9bq-MuiSvgIcon-root"
              data-testid="YouTubeIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M10 15l5.19-3L10 9v6m11.56-7.83c.13.47.22 1.1.28 1.9.07.8.1 1.49.1 2.09L22 12c0 2.19-.16 3.8-.44 4.83-.25.9-.83 1.48-1.73 1.73-.47.13-1.33.22-2.65.28-1.3.07-2.49.1-3.59.1L12 19c-4.19 0-6.8-.16-7.83-.44-.9-.25-1.48-.83-1.73-1.73-.13-.47-.22-1.1-.28-1.9-.07-.8-.1-1.49-.1-2.09L2 12c0-2.19.16-3.8.44-4.83.25-.9.83-1.48 1.73-1.73.47-.13 1.33-.22 2.65-.28 1.3-.07 2.49-.1 3.59-.1L12 5c4.19 0 6.8.16 7.83.44.9.25 1.48.83 1.73 1.73z"
              />
            </svg>
          </button>
        </a>
        <a
          aria-label="GitHub"
          class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone css-18rqtra-MuiTypography-root-MuiLink-root"
          data-mui-internal-clone-element="true"
          href="https://github.com/vimalmenon"
          target="_blank"
        >
          <button
            class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-53g0n7-MuiButtonBase-root-MuiIconButton-root"
            tabindex="0"
            type="button"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1umw9bq-MuiSvgIcon-root"
              data-testid="GitHubIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M12 1.27a11 11 0 00-3.48 21.46c.55.09.73-.28.73-.55v-1.84c-3.03.64-3.67-1.46-3.67-1.46-.55-1.29-1.28-1.65-1.28-1.65-.92-.65.1-.65.1-.65 1.1 0 1.73 1.1 1.73 1.1.92 1.65 2.57 1.2 3.21.92a2 2 0 01.64-1.47c-2.47-.27-5.04-1.19-5.04-5.5 0-1.1.46-2.1 1.2-2.84a3.76 3.76 0 010-2.93s.91-.28 3.11 1.1c1.8-.49 3.7-.49 5.5 0 2.1-1.38 3.02-1.1 3.02-1.1a3.76 3.76 0 010 2.93c.83.74 1.2 1.74 1.2 2.94 0 4.21-2.57 5.13-5.04 5.4.45.37.82.92.82 2.02v3.03c0 .27.1.64.73.55A11 11 0 0012 1.27"
              />
            </svg>
          </button>
        </a>
        <a
          aria-label="LinkedIn"
          class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone css-18rqtra-MuiTypography-root-MuiLink-root"
          data-mui-internal-clone-element="true"
          href="https://www.linkedin.com/in/real-vimal-menon/"
          target="_blank"
        >
          <button
            class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-53g0n7-MuiButtonBase-root-MuiIconButton-root"
            tabindex="0"
            type="button"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1umw9bq-MuiSvgIcon-root"
              data-testid="LinkedInIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M19 3a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h14m-.5 15.5v-5.3a3.26 3.26 0 0 0-3.26-3.26c-.85 0-1.84.52-2.32 1.3v-1.11h-2.79v8.37h2.79v-4.93c0-.77.62-1.4 1.39-1.4a1.4 1.4 0 0 1 1.4 1.4v4.93h2.79M6.88 8.56a1.68 1.68 0 0 0 1.68-1.68c0-.93-.75-1.69-1.68-1.69a1.69 1.69 0 0 0-1.69 1.69c0 .93.76 1.68 1.69 1.68m1.39 9.94v-8.37H5.5v8.37h2.77z"
              />
            </svg>
          </button>
        </a>
        <a
          aria-label="Resume"
          class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone css-18rqtra-MuiTypography-root-MuiLink-root"
          data-mui-internal-clone-element="true"
          href="https://vimalmenon.com/vimal-menon.pdf"
          target="_blank"
        >
          <button
            class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-53g0n7-MuiButtonBase-root-MuiIconButton-root"
            tabindex="0"
            type="button"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1umw9bq-MuiSvgIcon-root"
              data-testid="FileDownloadIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M19 9h-4V3H9v6H5l7 7zM5 18v2h14v-2z"
              />
            </svg>
          </button>
        </a>
      </div>
    </div>
  </footer>
</div>
`;
